### PR TITLE
Phase 1 client sprint

### PR DESCRIPTION
**Pull request by ranweiler**
Includes: #198, #191, #221, #253, #256, #267, #276, #326.

### Feature walkthrough
DOS setup:
- [x] Log in as DOS admin.
- [x] Enter risk limilts.
- [x] Submit risk limilts.
- [x] Enter RNG seed.
- [x] Submit RNG seed.
- [x] Review audit data. (#308)

County upload:
- [x] Log in as County Admin.
- [x] Select ballot manifest.
- [x] Enter ballot manifest hash.
- [x] Upload ballot manifest.
- [x] Select CVR export.
- [x] Enter CVR export hash.
- [x] Upload CVR export.

DOS select contests:
- [x] Log in as DOS admin.
- [x] Select contests for audit.
- [x] Submit contests for audit.
- [x] Publish ballots to audit.

County audit loop:
- [x] Log in as County Admin.
- [x] Begin audit.
- [x] Enter audit board info.
- [x] Submit audit board info.
- [x] Enter marks observed on ballot.
  - [x] Fetch contests for county
  - [x] Fetch contests under audit  
  - [x] Fetch ballots to audit
  - [x] Fetch ID of current ballot   
  - [x] Fetch full CVR of current ballot
  - [x] Plumb CVR data through forms
    - [x] Fix instructions form
    - [x] Add actions to update ACVR form state in local store
    - [x] Render ACVR forms
    - [x] Render ACVR marks from store ACVR form state
    - [x] Render review page from local ACVR form state
- [x] Submit marks observed on ballot.
- [x] Add "Ballot Card Not Found" button
- [x] Add label on hash code text fields
- [x] Observe county-level audit progress on dashboard

DOS monitor audit progress:
- [x] Log in as DOS admin.
- [x] Observe audit progress.
  - [x] Server updates dashboard state (#300)
  - [x] Fetch audit progress data
  - [x] Render audit progress (#309)

### Various UI/UX bugs
- [ ] ~~#279~~ (low priority, address in phase 2)
- [x] #304 
- [x] #305
- [ ] #306 
- [x] #307
- [x] #308 
- [x] #310 (~~may be blocked on ASM work~~ approximated for now based on dashboard state) 
- [x] #312
- [x] #313
- [x] #314
- [x] #318
- [x] #319 
- [x] #321 
- [x] #322
- [ ] #326 (client work done, verify after minor server update, see issue)
- [x] #352

### CODE REVIEW

File: server/eclipse-project/script/init-db [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -eux -o pipefail
+readonly __dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+readonly cmd="psql -d corla -a -f "${__dir}/insert-test-creds.sql""
```
Comment (dmzimmerman) [inline server/eclipse-project/script/init-db]: I don't love the fact that this is a duplicate copy of the .psql from the `test` hierarchy. Means we have to keep them in sync.

Comment (dmzimmerman) [inline server/eclipse-project/script/init-db]: (and they will definitely have to change, once we make authentication more robust)

Comment (dmzimmerman) [inline server/eclipse-project/script/init-db]: I don't love the fact that this is a duplicate copy of the .psql from the test hierarchy. Means we have to keep them in sync.
(and they will definitely have to change, once we make authentication more robust)

Comment (ranweiler) [inline server/eclipse-project/script/init-db]: Let's fix this in another PR, please.

**Pull request merged**